test/fuzz/lua/test_engine.lua

```diff
@@ -1429,6 +1447,7 @@ local function run_test(num_workers, test_duration, test_dir,
     table.insert(workers, errinj_f)
 
     local error_messages = {}
+    disable_errinj(errinj_set, space)
```

Comment:
I think you should stop the error injection fiber before disabling error injections, otherwise it may enable them back.

Reply:
Right, fixed:

```diff
--- a/test/fuzz/lua/test_engine.lua
+++ b/test/fuzz/lua/test_engine.lua
@@ -1447,7 +1447,6 @@ local function run_test(num_workers, test_duration, test_dir,
     table.insert(workers, errinj_f)
 
     local error_messages = {}
-    disable_errinj(errinj_set, space)
     for _, fb in ipairs(workers) do
         local ok, res = fiber.join(fb)
         if not ok then
@@ -1463,6 +1462,7 @@ local function run_test(num_workers, test_duration, test_dir,
             end
         end
     end
+    disable_errinj(errinj_set, space)
 
     teardown(space)
 ```